src/app/entity-groups/journal-entities/item-list-elements/search-result-list-elements/journal-issue/journal-issue-search-result-list-element.component.spec.ts

```diff
@@ -70,12 +104,65 @@ const enviromentNoThumbs = {
   }
 };
 
+const supervisionOrderDataService: any = jasmine.createSpyObj('supervisionOrderDataService', {
+  searchByItem: jasmine.createSpy('searchByItem'),
+});
+
+const supervisionOrder: any = {
```

Comment:
I forgot to mention in my review... I'm not understanding why all these `*-search-result-list-element.component.spec.ts` files had to be updated to include Supervision orders?  It appears you had to update every entity type's search results specs...but this looks really odd to me since you didn't modify these components.  Was this a mistake?  Or is there a reason why supervision orders have to now be included in all these tests?